Automattic/jetpack
# CRM Automations: Add workflow table

Fixes https://github.com/Automattic/zero-bs-crm/issues/3210

## Proposed changes:


This PR adds a table component to display many workflows in a table.

### Other information:


## Jetpack product discussion


NtYgueI8oAgiWGHTbHpnjh-fi-1_29

## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. There is no functionality yet, so primarily review the code.
2. Put `add_filter( 'jetpack_crm_feature_flag_automations', '__return_true' );` in an mu-plugin.
3. Visit `/wp-admin/admin.php?page=jpcrm-automations#/automations/test-workflow-table`. Verify that the visuals match the figma designs.
